src/main/kotlin/com/moshbit/katerbase/MongoHelper.kt

```diff
@@ -406,3 +406,12 @@ class AggregationPipeline {
     }
   }
 }
+
+class QueryStats(
```

Comment:
Pls add the other fields from executionStats https://docs.mongodb.com/manual/reference/explain-results/#std-label-executionStats (at least by just passing the `executionStats` map and just pass it non-typesafe)